application/app/process/spawned_process.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class DetachProcess
+class SpawnedProcess
```

Comment:
Thinking about it, I would call this `ScriptLauncher` and change the main method to `launch_script`.
We can keep it in this folder or move it back to Services. I do not mind either way

Reply:
OK, let me refactor it